plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -282,7 +282,7 @@ private boolean validateAppEngineProjectDetails() {
     }
 
     String projectId = getAppEngineProjectId();
-    if (!AppEngineProjectIdValidator.validate(projectId)) {
+    if (!projectId.isEmpty() && !ProjectIdValidator.validate(projectId)) {
       setErrorMessage(MessageFormat.format("Illegal App Engine Project ID: {0}.", projectId));
```

Comment:
Agreed. Filed https://github.com/GoogleCloudPlatform/cloud-tools-for-eclipse/issues/730 since this is an existing issue.